code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -90,13 +90,31 @@
 				//End bloody footprints
 
 				S.step_action()
+	if(movement_type & GROUND)
+		dirt_buildup()
 
 /mob/living/carbon/human/Process_Spacemove(movement_dir = 0) //Temporary laziness thing. Will change to handles by species reee.
 	if(dna.species.space_move(src))
 		return TRUE
 	return ..()
 
-/mob/living/carbon/human/dirt_buildup(strength)
+/mob/living/carbon/human/proc/dirt_buildup(strength)
```

Comment:
shouldn't this also have the strength = 1

Reply:
yes